packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/src/Providers/TypeProvider.cs

```diff
@@ -357,14 +382,6 @@ public void Update(
             {
                 XmlDocs = xmlDocs;
             }
-            if (modifiers != null)
```

Comment:
It looks like we may be going in the other direction here and instead using the Update methods. No need to fix in your PR, but we shouldn't remove anything from Update.

Reply:
I added them in https://github.com/microsoft/typespec/pull/5432 for the purpose of ResourceDataProvider.
Think it would be cleaner to remove them since not needed any more.

Reply:
We will end up needing them for every property so it would be less work in the future if we just leave them as is.

Reply:
reverted